readme.md

```diff
@@ -2,7 +2,16 @@
 
 <sup>(browser + oyster)</sup>
 
-[WIP] A monorepo of testing tools
+<p align="center">
+  <a href="https://github.com/fingerprintjs/broyster/actions/workflows/test.yml">
+    <img src="https://github.com/fingerprintjs/broyster/actions/workflows/test.yml/badge.svg" alt="Build status">
+  </a>
+  <a href="https://www.npmjs.com/package/@fpjs-incubator/broyster">
+    <img src="https://img.shields.io/npm/v/@fpjs-incubator/broyster.svg" alt="Current NPM version">
+  </a>
+</p>
+
+A monorepo of testing tools
 
 Project structure:
```

Comment:
Consider adding a simple explanation of what exactly the project does, what is the common use case etc. :)